pyproject.toml

```diff
@@ -80,3 +80,31 @@ packages = [
     "h5py.tests.data_files",
     "h5py.tests.test_vds",
 ]
+
+[tool.cibuildwheel]
+build-verbosity = 3
+manylinux-x86_64-image = "ghcr.io/h5py/manylinux2014_x86_64-hdf5"
+manylinux-aarch64-image = "ghcr.io/h5py/manylinux2014_aarch64-hdf5"
+test-requires = [
+    "tox",
+    "codecov",
+    "coverage",
+]
+test-command = "bash {project}/ci/cibw_test_command.sh {project} {wheel}"
+
+[tool.cibuildwheel.linux]
+environment-pass = ["GITHUB_ACTIONS"]
+
+[tool.cibuildwheel.linux.environment]
+CFLAGS = "-g1"
+
+[tool.cibuildwheel.macos]
+# https://cibuildwheel.pypa.io/en/stable/faq/#macos-passing-dyld_library_path-to-delocate
+repair-wheel-command = """\
+DYLD_FALLBACK_LIBRARY_PATH=$HDF5_DIR/lib \
+delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel} \
+"""
+
+[tool.cibuildwheel.macos.environment]
+H5PY_ROS3 = "0"
+H5PY_DIRECT_VFD = "0"
```

Comment:
I was trying to avoid changing behavior / use of env vars where possible, and the only place this seems to be set in `master` is in the macos wheels

https://github.com/h5py/h5py/blob/cc468545218b2465342f1f94f14103ac2848a682/azure-pipelines.yml#L52-L59

Reply:
Interesting, we should probably check the reported configuration of all the wheels once this PR is ready, and ensure it's what should go onto PyPI.

Reply:
I'm happy to cross-check first the set of artifacts produced in this PR with those on PyPI -- I can at least do `import`-based checks on macOS arm64, Linux x86_64, and Windows. What functions/commands/files should I check to see the configuration used to build the wheel?

Reply:
... ahh, `h5py.get_config()` has some stuff I can use!